src/lib.rs

```diff
@@ -109,12 +261,81 @@ impl XenControl {
         last_error!(self, ())
     }
 
+    pub fn monitor_software_breakpoint(&self, domid: u32, enable: bool) -> Result<()> {
+        let xc = self.handle.as_ptr();
+        (self.libxenctrl.clear_last_error)(xc);
+        let rc = (self.libxenctrl.monitor_software_breakpoint)(xc, domid, enable);
+        if rc < 0 {
+            println!("last OS error: {:?}", Error::last_os_error());
+        }
+        last_error!(self, ())
+    }
+
+    pub fn monitor_mov_to_msr(&self, domid: u32, msr: u32, enable: bool) -> Result<()> {
+        let xc = self.handle.as_ptr();
+        (self.libxenctrl.clear_last_error)(xc);
+        let rc = (self.libxenctrl.monitor_mov_to_msr)(xc, domid.try_into().unwrap(), msr, enable);
+        if rc < 0 {
+            println!("last OS error: {:?}", Error::last_os_error());
```

Comment:
you should print an error message.
return and Error:
https://github.com/Wenzel/xenstore/blob/master/src/lib.rs#L38